pkg/config/system_probe.go

```diff
@@ -197,6 +197,8 @@ func InitSystemProbeConfig(cfg Config) {
 
 	cfg.BindEnvAndSetDefault(join(spNS, "language_detection.enabled"), false)
 
+	cfg.BindEnvAndSetDefault(join(spNS, "process_service_inference.enabled"), false, "DD_SYSTEM_PROBE_PROCESS_SERVICE_INFERENCE_ENABLED")
```

Comment:
Do we want to default this to `true`?

Reply:
I guess for NPM this would make sense to default to `true`. Talking with @guyarb it was set to false since this is only used in `usm` for non-containerized envs, but for our case it would be good to have these anyway I think. Should I maybe finish the backend before enabling? Or do you think I should enable even if the effects are not yet finished in the backend?

Reply:
If we default it off now it would likely ship in the next agent version that way.

Is it harmful to have it on? I don't know what it entails

Reply:
I'm OK with leaving this for a separate PR but I would like to have this resolved before the agent release/freeze.

The feature that we're building around this relies on this behavior to be enabled and it not be very helpful if customers have to manually enable this configuration

Reply:
Do we want to enable the backend portion before enabling this by default and testing it has what we need? We may have to go back and modify/enhance this if necessary.